src/components/ThreejsEditorModal.jsx

```diff
@@ -117,7 +112,14 @@ export class ThreejsEditorModal extends ModalDialog {
         const toolbar = new Toolbar(this.editor);
         this.domElement.appendChild(toolbar.dom);
 
-        this.initializeControls();
+        // initialize controls
+        this.editor.controls = new OrbitControls(this.editor.camera, viewport.dom);
+        this.editor.controls.up = new THREE.Vector3(0, 0, 1);
```

Comment:
Maybe move to `initializeControlsInEditor`

Reply:
Done